routes/auth_routes.py

```
@@ -0,0 +1,134 @@
+# coding: utf-8
+"""
+基于密码验证的认证路由与会话守卫。
+
+- ``register_auth_routes(registry)``：登录/登出/当前用户/改密 API。
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)